src/extension/content-script/onstart.ts

```diff
@@ -18,10 +19,14 @@ async function onstart() {
   injectScript(browser.runtime.getURL("js/inpageScriptWebBTC.bundle.js"));
 
   // window.nostr
-  const nostrEnabled = (await api.getAccount()).nostrEnabled;
-  if (nostrEnabled) {
+  if (accountDetails.nostrEnabled) {
     injectScript(browser.runtime.getURL("js/inpageScriptNostr.bundle.js"));
   }
+
+  // window.secp256k1
```

Comment:
should this be window.liquid?

Reply:
Ah, this is decided by Bumi and the person from Ionio (Tiero?) // @bumi

Reply:
But it will always use the liquid key? won't webbtc also have signSchnorr?